activerecord/lib/active_record/associations.rb

```diff
@@ -1853,6 +1863,10 @@ def has_one(name, scope = nil, **options)
         # [:ensuring_owner_was]
         #   Specifies an instance method to be called on the owner. The method must return true in order for the
         #   associated records to be deleted in a background job.
+        # [:query_constraints]
+        #   Serves as a composite foreign key. Defines the list of columns to be used to query the associated object.
+        #   This is an optional option. By default Rails will attempt to derive the value automatically.
+        #   When the value is set the Array size must match associated model's primary key or `query_constraints` size.
```

Comment:
@nvasilevski This should be `+query_constraints+` since RDoc doesn't parse the backticks as code, FYI

Reply:
Thank you! Will change it in https://github.com/rails/rails/pull/49407

Reply:
Thanks for fixing it and all your hard work :bow: